src/makeCalculator.js

```diff
@@ -37,7 +37,72 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const allOperates = {
+
+    // змінна в якій буде зберігатись результат
+    resultOperation: 0,
+
+    // геттер який повертає результат , тобто нашу змінну з результатом
+    get result() {
+      return this.resultOperation;
+    },
+
+    // геттер який повертає наш об'єкт,
+    // для того щоб можна було використовувати чейнінг
+    get thisObject() {
+      return this;
+    },
+
+    /**
+     * @param {number} number
+     * сеттер - який буде змінювати нашу результуючу змінну
+     */
+    set changeResult(number) {
+      this.resultOperation = number;
+    },
+
+    // дана функція приймає (callback) і число (number) - для маніпуляцій з ним
+    operate(callback, number) {
+      this.changeResult = callback(this.resultOperation, number);
+
+      return this.thisObject;
+    },
```

Comment:
For better understanding `this` I recommend you to rewrite operate (other code you need will change too): 

```suggestion
    operate(operation, value) {
      operation.call(this, value);

      return this;
    },
```